docs: update doc links in usage doc (#465)
kubernetes-sigs/secrets-store-csi-driver: docs/book/src/getting-started/usage.md

## [OPTIONAL] Sync with Kubernetes Secrets

> Refer to [Sync as Kubernetes Secret](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.

Reviewer: ```suggestion
Refer to [Sync as Kubernetes Secret](../topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
```

We only need the relative path for the docs website to redirect to correct page.

Author: sure, updated.

### [OPTIONAL] Set ENV VAR

> Refer to [Set as ENV var](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/set-as-env-var.md) for steps on syncing the secrets-store content as Kubernetes secret and using the secret for env variables in the deployment.

Reviewer: ```suggestion
Refer to [Set as ENV var](..topics/set-as-env-var.md) for steps on syncing the secrets-store content as Kubernetes secret and using the secret for env variables in the deployment.
```

## [OPTIONAL] Enable Auto Rotation of Secrets

> You can setup the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [Secret Auto Rotation](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/secret-auto-rotation.md) for steps on enabling auto rotation.

Reviewer: ```suggestion
You can set up the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [Secret Auto Rotation](../topics/secret-auto-rotation.md) for steps on enabling auto rotation.
```
